sodetlib/operations/uxm_relock.py

```diff
@@ -260,7 +260,7 @@ def plot_channel_resonance(S, cfg, band, chan):
 
 @sdl.set_action()
 def uxm_relock(
-    S: SmurfControl, cfg, bands=None, show_plots=False,
+    S, cfg, bands=None, show_plots=False,
```

Comment:
This seems unrelated to this PR but ok…

Reply:
This was required for me to get docs to build... I may invest some time cleaning up the docs and making sure everything is up to date.